python/oneflow/nn/modules/adaptive_pool.py

```diff
@@ -64,25 +64,24 @@ class AdaptiveAvgPool1d(Module):
 
     """
 
-    def __init__(self, output_size) -> None:
+    def __init__(self, output_size: _size_1_t) -> None:
         super().__init__()
-        self.output_size = output_size
+        assert output_size is not None
+        self.output_size = _single(output_size)
 
     def forward(self, x):
         assert len(x.shape) == 3
-        if isinstance(self.output_size, tuple):
-            new_output_size = self.output_size[0]
-        elif isinstance(self.output_size, int):
-            new_output_size = self.output_size
-        else:
-            raise ValueError("'output_size' should be integer or tuple")
-        return flow.F.adaptive_avg_pool1d(x, output_size=(new_output_size,))
+        assert len(self.output_size) == 1,  f"the length of 'output_size' does not match the input size, 1 expected"
```

Comment:
第一个assert和第二个assert可以合并
```suggestion
        assert len(x.shape) == 3 and assert len(self.output_size) == 1,  f"the length of 'output_size' does not match the input size, 1 expected"
```